.github/workflows/trunk.yml

```diff
@@ -127,6 +127,25 @@ jobs:
       ANDROID_SIGN_PASS: ${{ secrets.ANDROID_SIGN_PASS }}
       SCRIBE_GRAPHQL_ACCESS_TOKEN: ${{ secrets.SCRIBE_GRAPHQL_ACCESS_TOKEN }}
 
+  linux-bionic-py3_7-clang9-slow-build:
+    name: linux-bionic-py3.7-clang9-slow
+    uses: pytorch/pytorch/.github/workflows/_linux-build.yml@master
+    with:
+      build-environment: linux-bionic-py3.7-clang9-slow
+      docker-image-name: pytorch-linux-bionic-py3.7-clang9
```

Comment:
Do you need a separate build because there are no trunk-only builds?
@suo with new lint workflow is there a way to add a trunk-only test-shard to PR workflow?

Reply:
it's just standard GH stuff now, so a job with an if statements would work.

Although: can we not just use one of the many trunk gcc builds instead? Is there something about clang that is special here

Reply:
@suo how would the job with if statements work? e.g. where to put if statements?
re: gcc builds, I think we just want a cpu-only build. AFAICT all the other builds in trunk.yml are for cuda (other than parallelnative, which seems like a special configuration).

Reply:
oh, that's interesting. Yeah it looks like there is no trunk-only CPU build. idk, personally I think this change is fine; I'd rather just do an extra build than make the workflows more marginally more complicated.

Reply:
for reference, it would look like the same thing you have, with one additional line: 
```
  if: github.event == 'push'
```

But like I said, I feel like we should just keep pull jobs in pull.yml, and trunk jobs in trunk.yml. CPU builds take sub-10m and are super cheap, so it's not a big deal to overbuild a bit